### PR TITLE
10176 - Pending Motions Table Feedback Fix

### PR DESCRIPTION
**Pull request by cruzjone-flexion**

### CODE REVIEW

File: shared/src/business/useCases/pendingMotion/deleteDocketEntryWorksheetInteractor.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export const deleteDocketEntryWorksheetInteractor = async (
+  applicationContext: IApplicationContext,
+  docketEntryId: string,
+): Promise<void> => {
+  // TODO: Determine who can delete Docket Entry Worksheet
```
Comment (nechama-krigsman) [inline shared/src/business/useCases/pendingMotion/deleteDocketEntryWorksheetInteractor.ts]: Does this need to be included in this PR?

**Review by nechama-krigsman — APPROVED** [review]

File: shared/src/business/useCases/pendingMotion/getPendingMotionDocketEntriesForCurrentJudgeInteractor.ts [hunk 2]
```diff
@@ -11,78 +11,63 @@ import {
...
-    await attachDocketEntryWorkSheets(applicationContext, batchCases);
-
-  const uniquePendingMotions: DocketEntryWithWorksheet[] =
-    removeDuplicateDocketEntries(pendingMotionsDocketEntriesWithWorksheet);
+            currentDate,
+          ),
+      ),
+    );
+
+  const removeMotionsThatHaveBeenHandledInDynamo = de => de.pending;
```
Comment (nechama-krigsman) [inline shared/src/business/useCases/pendingMotion/getPendingMotionDocketEntriesForCurrentJudgeInteractor.ts:60]: Can this be handled in `getLatestDataForPendingMotionsFromDynamo` and only return the pending motions?

File: shared/src/business/useCases/pendingMotion/deleteDocketEntryWorksheetInteractor.ts (hunk 1 above)
Comment (cruzjone-flexion) [reply shared/src/business/useCases/pendingMotion/deleteDocketEntryWorksheetInteractor.ts]: Completed

File: shared/src/business/useCases/pendingMotion/getPendingMotionDocketEntriesForCurrentJudgeInteractor.ts (hunk 2 above)
Comment (cruzjone-flexion) [reply shared/src/business/useCases/pendingMotion/getPendingMotionDocketEntriesForCurrentJudgeInteractor.ts:60]: We could add it but then that could potentially hide an important step in the method.

**Pull request merged**